src/Compilers/VisualBasic/Portable/Language/LanguageVersion.vb

```diff
@@ -173,5 +169,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Overrides Function ToString() As String
             Return Version.ToDisplayString()
         End Function
+        Shared Function [For](feature As Feature) As VisualBasicRequiredLanguageVersion
```

Comment:
Nit: missing empty line above.
Also, I'd probably have added a constructor instead of a factory method. But that's not essential.

Reply:
#Done